### ISSUE TITLE
Remove Cargo authors line

### ISSUE THREAD

**Issue by notmandatory** [bitcoindevkit/bdk-ffi#407]
In a future PR I want to remove this line and replace it with:
```toml
homepage = "https://bitcoindevkit.org"
repository = "https://github.com/bitcoindevkit/bdk"
```

_Originally posted by @notmandatory in https://github.com/bitcoindevkit/bdk-ffi/pull/402#discussion_r1368925254_

**Comment by notmandatory** [comment]
@artfuldev since we have alot of new devs I want to remove the authors line from the Cargo.toml and other parts of the projects to replace with "Bitcoin Dev Kit Developers".  Since it's only you and I as "authors" now I want to confirm, do you have any concerns making this change?

**Comment by artfuldev** [comment]
Please go ahead, it's amazing to have all these new devs! 🙂

**Comment by thunderbiscuit** [comment]
Merged with #425!
